cookiecutter.json

```diff
@@ -1,8 +1,31 @@
 {
   "plugin_name": "myplugin",
-  "package_name": "tutor-contrib-{{ cookiecutter.plugin_name }}",
-  "module_name": "tutor{{ cookiecutter.plugin_name.lower().replace('-', '_') }}",
-  "git_repo": "https://github.com/myusername/{{cookiecutter.package_name }}",
+  "package_name": "tutor-contrib-{{ cookiecutter.plugin_name.lower() | trim }}",
+  "module_name": "tutor{{ cookiecutter.plugin_name.lower().replace('-', '_') | trim }}",
+  "description": "{{ cookiecutter.plugin_name}} plugin for Tutor",
+  "git_username": "myusername",
+  "git_repo": "https://github.com/{{ cookiecutter.git_username | trim }}/{{ cookiecutter.package_name | trim }}",
   "author": "John Doe",
-  "license": "AGPLv3"
+  "email": "john.doe@example.com",
+  "tutor_version": "17.0.0",
+  "version": "0.0.1",
+  "license": ["AGPLv3", "Apache 2.0", "BSDv3", "MIT", "Not open source"],
+  "init_git": "y",
+  "include_ci": "y",
+  "__prompts__": {
+    "plugin_name": "Plugin name:",
+    "package_name": "Package name:",
+    "module_name": "Module name:",
+    "description": "Short description about the plugin:",
+    "git_username": "Your GitHub username or organization:",
+    "git_repo": "GitHub repository URL:",
+    "author": "Author's full name:",
+    "email": "Author's email:",
+    "tutor_version": "Tutor version:",
+    "version": "Plugin version:",
+    "license": "Select the license for your package:",
+    "init_git": "Initialize a git repository (y/n):",
+    "include_ci": "Include CI files (y/n):",
+    "_extensions": ["extensions.version_range"]
```

Comment:
Sure.

Reply:
I forgot to load this extension inside `cookiecutter.json`. My bad.
we can live without that, but it will make tutor version section dirty and hard to read.